### ISSUE TITLE
Unwarranted complains with  FOR .. IN EXECUTE ..

### ISSUE THREAD

**Issue by johto** [okbob/plpgsql_check#26]
The following function:

```SQL
create function out1(OUT f1 int, OUT f2 int)
returns setof record as
$$
begin
    for f1, f2 in
        execute $q$ select 1, 2 $q$
    loop
        return next;
    end loop;
    end
$$ language plpgsql;
```

is being complained about as follows:

```
=# select * from plpgsql_check_function('out1()');
                plpgsql_check_function
------------------------------------------------------
 warning:00000:1:DECLARE:unused variable "*internal*"
 warning extra:00000:unmodified OUT variable "$1"
 warning extra:00000:unmodified OUT variable "$2"
(3 rows)
```

**Comment by okbob** [comment]
Thank you for report.
